### PR TITLE
Add launcher script

### PR DESCRIPTION
**Pull request by drasko**
Adds script to launch all EdgeX Go services

Signed-off-by: Drasko DRASKOVIC <drasko.draskovic@gmail.com>

### CODE REVIEW

File: bin/edgex-launch.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```
Comment (feclare) [inline bin/edgex-launch.sh]: Add license header

File: bin/edgex-launch.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+###
+# Launches all EdgeX Go binaries (must be previously built).
+#
+# Expects that Consul is already installed and running.
```
Comment (feclare) [inline bin/edgex-launch.sh]: Add comment about mongo, that it is also needed to be running

File: bin/edgex-launch.sh (hunk 1 above)
Comment (drasko) [reply bin/edgex-launch.sh]: Added

**Comment by drasko** [comment]
@janko-isidorovic this one can be merged.

**Pull request merged**